website/docs/cdktf/release/upgrade-guide-v0-15.mdx

```diff
@@ -91,3 +91,16 @@ func NewExampleCdktfGoAwsStack(scope constructs.Construct, id string) cdktf.Terr
 	return stack
 }
 ```
+
+### Deprecation: We deprecate the `context` field of the `cdktf.json`
```

Comment:
In theory the context field can be used for more than feature flags (it allows to pass any kind of context to the `App`) – maybe reword this to denote that we intend to deprecate the feature flags, not all of `context`.

Reply:
Related issues:
https://github.com/hashicorp/terraform-cdk/issues/347
https://github.com/hashicorp/terraform-cdk/issues/2019

(I'm open to removing support for `context`, but so far we've only agreed to discontinue the feature flags)